### DISCUSSION TITLE
Explanation of terms - home page

### DISCUSSION THREAD

**Discussion by spiderknight** [pierreaubert/spinorama#245] [General]
Hi. I looked around but couldn't find def'n of:

- Tonality
- Flatness

Thanks.

**Comment by pierreaubert** [comment]
Hello,

I will add the defintion to the Help page.

- Tonality: Harman score based on  CEA2034 (computed [here](https://github.com/pierreaubert/spinorama/blob/abec58b8f2f93251d42201b753d6ed25a3276a1d/src/spinorama/compute_scores.py#L214))
- Flatness: deviation from flat for the on-axis response (computed [here](https://github.com/pierreaubert/spinorama/blob/abec58b8f2f93251d42201b753d6ed25a3276a1d/src/spinorama/compute_estimates.py#L80))

You want the tonality score to be as high as possible and the flatness to be as close to 0 as possible.

**Comment by pierreaubert** [comment]
I added documentation in the help page and links from the homepage to find it easily.
